multihost_test/Sanity/conftest.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+import subprocess
+import pytest
+import time
+import ldap
+import random
+from pytest_multihost import make_multihost_fixture
+from sssd.testlib.common.qe_class import session_multihost
+from sssd.testlib.common.paths import SSSD_DEFAULT_CONF
+from constants import ds_instance_name, ds_suffix, krb_realm
+from sssd.testlib.common.utils import PkiTools, sssdTools, LdapOperations
+from sssd.testlib.common.libdirsrv import DirSrvWrap
+from sssd.testlib.common.exceptions import PkiLibException, LdapException
+from sssd.testlib.common.libkrb5 import krb5srv
+
+
+def pytest_configure():
+    """ Namespace hook to add below dict in the pytest namespace """
+    pytest.num_masters = 1
```

Comment:
At the moment these tests are only on a single host, do we need to provision this?

Reply:
Done